src/main/java/javapoz24/team3/rental/application/api/CarController.java

```diff
@@ -0,0 +1,42 @@
+package javapoz24.team3.rental.application.api;
+
+import javapoz24.team3.rental.application.CarService;
+import javapoz24.team3.rental.domain.car.Car;
+import javapoz24.team3.rental.domain.car.CarDTO;
+import javapoz24.team3.rental.domain.car.CarDomainService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/cars")
+public class CarController {
+
+    private final CarService carService;
+
+    public CarController(CarService carService) {
+        this.carService = carService;
+    }
+
+    @GetMapping
+    public List<CarDTO> getAll() {
+        return carService.getAll();
+    }
+
+    @GetMapping("/{id}")
+    public CarDTO getById(@PathVariable(name = "id") Long id) {
+        Optional<Car> optCar = carService.getById(id);
+        return optCar.map(CarDTO::fromCar).orElse(null);
```

Comment:
ja bym to zapisał tak:
return carService.getById(id)
                       .map(CarDTO::fromCar)
                       .orElse(null);